src/mpid/ch4/src/ch4_progress.h

```diff
@@ -33,6 +57,11 @@
 
 extern MPL_TLS int global_vci_poll_count;
 
+/* Counter to track how many consecutive calls to MPIDI_progress_test fail to make progress.
+ * Used when MPIR_CVAR_CH4_PROGRESS_THROTTLE=true.
+ */
+MPL_TLS int no_progress_counter = 0;
```

Comment:
You can't define global variables in a header file (ch4_progress.h). Define it the same way as `global_vci_poll_count`

Reply:
Fixed, now declared the same as `global_vci_poll_count`